### PR TITLE
feat: Add List::Value component to rose addon

### PR DESCRIPTION
**Pull request by gsusmi**
Introduces a new ~~`Rose::List::Value`~~ `Rose::Form::List` component based on `Rose::List::KeyValue`.
rose-list-value

### CODE REVIEW

File: addons/rose/addon/components/rose/list/value/item/index.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<li class="rose-list-value-item" id={{concat "item-value-" this.id}}>
```
Comment (gsusmi) [inline addons/rose/addon/components/rose/list/value/item/index.hbs:1]: Struggled a bit here to find an aria label to add. Ended up with using an unique id coz there isn't a label within this component, unless a `title` or `label` can be passed in. Thoughts?

**Review by randallmorey — CHANGES_REQUESTED** [review]
Sorry to keep bouncing around on the naming.  At its core, this is not a form-specific list.  Can this be rethought?

I'd also like to see a bit more description in the story that explains the separated appearance, as well as the way that `cell` works.

File: addons/rose/addon/components/rose/form/list/index.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ul class="rose-form-list"
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/form/list/index.hbs:1]: Should this component go away?

File: addons/rose/addon/components/rose/list/separated/index.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+<Meta title="Rose|Form/List" component="Form/List" />
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/list/separated/index.stories.mdx:5]: Naming.

File: addons/rose/addon/components/rose/list/separated/index.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+# List
+
+Renders an unordered list in form.
+
+```html
+<Rose::Form::List @label="list-example" as |list|>
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/list/separated/index.stories.mdx:12]: Naming.

File: addons/rose/addon/components/rose/list/separated/index.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      </item.cell>
+    </list.item>
+  </Rose::Form::Input>
+</Rose::Form::List>
+```
+
+<Preview>
+  <Story name="Basic" height="17rem">{{
+    template: hbs`
+    <Rose::Form::List @label="list-example" as |list|>
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/list/separated/index.stories.mdx:35]: Naming.

File: addons/rose/tests/dummy/app/templates/index.hbs [hunk 6]
```diff
@@ -1,93 +1,36 @@
...
-    <h1>Authentication Pending</h1>
-  </:header>
-  <:body>
-    <p>Complete authentication in the newly-opened window.</p>
-    <p>Don't see the new window?  <a href="#">Try again.</a></p>
-    <p><a href="#">Cancel</a></p>
-  </:body>
-</Rose::Dialog::Cover>
+<Rose::List::Separated as |list|>
+  <Rose::Form as |form|>
```
Comment (randallmorey) [inline addons/rose/tests/dummy/app/templates/index.hbs:2]: Minor point but I'd swap the order here.  Form should be the parent of list.

File: addons/rose/tests/integration/components/rose/form/list-test.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, findAll } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | rose/form/list', function (hooks) {
```
Comment (randallmorey) [inline addons/rose/tests/integration/components/rose/form/list-test.js:6]: Isn't the form list going away?

**Review by randallmorey — CHANGES_REQUESTED** [review]

**Comment by gsusmi** [comment]
Moving into draft. Apologies @randallmorey, this was still in-progress but I forgot to update PR.

**Comment by gsusmi** [comment]
Closing in favor of updating table component styles.

**Pull request closed without merging**